auction-server/src/server.rs

```diff
@@ -248,16 +257,22 @@ pub async fn start_server(run_options: RunOptions) -> anyhow::Result<()> {
     let wallet = run_options.subwallet_private_key.parse::<LocalWallet>()?;
     tracing::info!("Using wallet address: {:?}", wallet.address());
 
-    let chains = match setup_chain_store(config_map.clone(), wallet.clone()).await {
-        Ok(chain_store) => chain_store,
-        Err(err) => {
-            tracing::error!("Failed to set up chain store: {:?}", err);
-            return Err(err);
-        }
-    };
+    let chains = setup_chain_store(config_map.clone(), wallet.clone()).await?;
```

Comment:
again, maybe the naming here --> `evm` suffix

Reply:
since svm config is an option, we should probably also make evm config an option in the long run?